server/src/main/kotlin/suwayomi/tachidesk/manga/impl/MangaList.kt

```diff
@@ -27,6 +27,14 @@ object MangaList {
     }
 
     suspend fun getMangaList(sourceId: Long, pageNum: Int = 1, popular: Boolean): PagedMangaListDataClass {
+        /*
+        * Page < 0 is a invalid index in the Tachiyomi api,
+        * depending on the source, it can error or duplicate results to the next page.
+        * Hence, handling it explicitly below
+        * */
+        require(pageNum > 0) {
+            "Page 0 is an invalid index"
```

Comment:
`pageNum < 0 is invalid` or `pageNum = $pageNum is not in valid range` or something similar